test/AssetManager.js

```diff
@@ -158,12 +158,12 @@ describe('AssetManager', function () {
     });
 
     it('should not be able to update activity status of asset if assetID is 0', async function () {
-      const tx = assetManager.updateAssetStatus(0, 0);
```

Comment:
why is an integer "0" being passed instead of a `bool`?
`bool` should be passed for consistency

Reply:
+1

Reply:
replaced from numeric to appropriate boolean values @SkandaBhat @hrishikeshio